filebeat/scripts/generator/fields/main.go

```diff
@@ -95,20 +100,34 @@ func newFieldYml(name, typeName string, noDoc bool) *fieldYml {
 func newField(lp string) field {
 	lp = lp[1 : len(lp)-1]
 	ee := strings.Split(lp, ":")
```

Comment:
What stands `ee` for?

Reply:
Well, it's "elements". I don't really have a good name for elements of `SYNTAX:SEMANTIC:TYPE` split at `:`s. I can rename it to `elements`, but I don't think it would help. :(

Reply:
I quite like `elements`, sounds better to me then `ee`